README.md

```diff
@@ -34,6 +34,7 @@ The following plugins have been added to help better illustrate Backstage featur
 - [Home](https://demo.backstage.io/home)
 - [Tech Radar](https://demo.backstage.io/tech-radar)
 - [To Do](https://demo.backstage.io/catalog/default/component/backstage-demo/todos)
+- [Notifications](https://demo.backstage.io/notifications)
```

Comment:
Can you create an Alpha subheading in this section? As Notifications is pretty new I think that makes more sense to me. Can we also perhaps include a link to the related BEP?